pyccl/emulators/baccoemu_nonlinear_pk.py

```diff
@@ -0,0 +1,85 @@
+__all__ = ("BaccoemuNonlinear",)
```

Comment:
You could house all three classes (parent and children) in the same `.py` file

Reply:
The nonlinear has a different __init__, of course we can override everything but since it's only 2 classes and each would override 2 of the 4 methods it has... maybe we can keep them separate? Anyway, not a strong opinion, if you feel this would be better I'll change it.